www-cronrobot/Gemfile

```diff
@@ -64,7 +64,7 @@ group :development do
 end
 
 group :production do
-  gem 'pg'
+ gem 'pg'
 end
```

Comment:
the text color is not great here

typing regular html such as `<p>hello</p>, <ul>, `, etc, should work on all pages

![image](https://user-images.githubusercontent.com/8007722/132593927-1b96d7c5-af3e-429f-83fb-45eb53f9245d.png)